util/module.h

```diff
@@ -520,6 +520,7 @@ struct module_env {
 	struct edns_known_option* edns_known_options;
 	/* Number of known edns options */
 	size_t edns_known_options_num;
+	struct edns_tags* edns_tags;
```

Comment:
A comment line above the member, like the other entries.  `/** EDNS client tag information */` or so.

Reply:
thanks, fixed